models/models.go

```diff
@@ -129,6 +129,7 @@ func init() {
 		new(OAuth2Application),
 		new(OAuth2AuthorizationCode),
 		new(OAuth2Grant),
+		new(Task),
```

Comment:
This needs to be in new db migration (`Sync2`)

Reply:
done.